app/file_sender.py

```diff
@@ -33,18 +34,20 @@ def process_complete_file(complete_partial_file: Path, pack_code: PackCode, cont
 
     try:
         copy_files_to_sftp(temporary_files_paths, SUPPLIER_TO_SFTP_DIRECTORY[supplier])
-
     except Exception as ex:
         context_logger.error('Failed to send files to SFTP', file_paths=list(map(str, temporary_files_paths)))
         context_logger.warn('Deleting failed encrypted and manifest print files',
                             file_paths=list(map(str, temporary_files_paths)))
         delete_local_files(temporary_files_paths)
         raise ex
 
-    # TODO upload encrypted print file and manifest to GCS
-
     context_logger.info('Successfully sent print files to SFTP', file_paths=list(map(str, temporary_files_paths)))
-    temporary_files_paths.append(complete_partial_file)
+
+    context_logger.info('Deleting partial files', file_paths=list(map(str, temporary_files_paths)))
```

Comment:
Log line will say it's deleting the encrypted print file and manifest which it isn't yet.